model/task/db.go

```diff
@@ -195,17 +195,19 @@ var (
 		},
 	}
 
-	updateDisplayTasksAndTasksExpression = bson.M{
+	updateDisplayTasksAndTasksSet = bson.M{
 		"$set": bson.M{
 			CanResetKey: true,
-		},
-		"$unset": bson.M{
-			AbortedKey:              "",
-			AbortInfoKey:            "",
-			OverrideDependenciesKey: "",
-		},
-		"$inc": bson.M{ExecutionKey: 1},
-	}
+			ExecutionKey: bson.M{
+				"$add": bson.A{"$" + ExecutionKey, 1},
+			},
+		}}
+	updateDisplayTasksAndTasksUnset = bson.M{
+		"$unset": bson.A{
+			AbortedKey,
+			AbortInfoKey,
+			OverrideDependenciesKey,
+		}}
```

Comment:
To check, is this change just to make it easier to chain together the multiple `$set` updates with adding the display status?

Reply:
Yes!